reporting.md

```diff
@@ -499,10 +497,9 @@ Figure: JSON Report Format
     Presently, the only three valid choices are `tlsa`, `sts`, and the literal
     string `no-policy-found`. It is provided as a string.
 * `policy-string`: The string serialization of the policy, whether TLSA record
-    or MTA-STS policy. Any linefeeds from the original policy MUST be replaced with
-    [SP]. TODO: Help with specifics.
+    or MTA-STS policy. Any linefeeds from the original policy MUST be replaced with space.
```

Comment:
For TLSA records, is it worth adding a reference to section 2.3 here (https://tools.ietf.org/html/rfc6698#section-2.3) for how to present it? 

I also do not understand why linefeeds are not valid; JSON strings can handle newline characters in strings (via "\n").